CSS/style.css

```diff
@@ -614,17 +514,64 @@ button:active {
     padding: 0 20px;
   }
 
+  .card-container .card-image img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  .works-section-title h2 {
+    padding: 80px 0 32px;
+  }
+
+  .works-section-title h2::after {
+    display: none;
+  }
+
   .scroll-down-icon {
     margin: 64px 0 245px;
   }
-}
 
-.cards-container {
-  display: grid;
-  grid-template-columns: repeat(1, 1fr);
-  grid-gap: 1.5rem;
-  width: 90%;
-  margin: 2rem  0;
+  .cards-container,
+  .frame-wrapper {
+    width: calc(654px + 24px + 20px);
+    padding: 10px 10px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 24px;
+    grid-auto-rows: 472px;
+    margin: 0 auto;
+    text-align: center;
+  }
```

Comment:
- [ ] Graet job adding some animations whatever you have some overflow when the animation makes effect :point_down: 
- [ ] 
![k2](https://user-images.githubusercontent.com/57562869/149513394-2e059d6d-ca13-4736-a57d-f0f0123a947c.png)

- Can you try to decrease your container height or increase the height of your frames to prevent that eg:
`grid-auto-rows: cals(height - 5px);`

Reply:
Now it's okay, I changed the `height` of `grid-auto-rows` :ok_hand: